app/tray/Notify/index.js

```diff
@@ -590,15 +554,6 @@ class Notify extends React.Component {
           {this.contractData()}
         </div>
       )
-    } else if (notify === 'hotAccountWarning') {
```

Comment:
how come we're removing this one?

Reply:
I don't think it's used?

Reply:
ok we only show this warning in the dash? that works then. also we should probably remove the "are in alpha" part of this warning

Reply:
Updated though the wording may need an update now. This notification appears to be triggered when adding keystore accounts only